am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/importer/ResourceDefinitionIntegrationTest.java

```diff
@@ -16,7 +16,7 @@
 
     @Test
     void shouldNotBeAbleToCreateResourceForServiceThatDoesNotExist() {
-        assertThatExceptionOfType(UnableToExecuteStatementException.class).isThrownBy(() ->
+        assertThatExceptionOfType(PersistenceException.class).isThrownBy(() ->
```

Comment:
I believe we have another test similar to this for roles. Would this also need to be updated to PersistenceException?

Reply:
The other test already expects `PersistenceException`.

```
assertThatExceptionOfType(PersistenceException.class)
            .isThrownBy(() -> service.grantDefaultPermission(createDefaultPermissionGrant(READ_PERMISSION)))
            .withMessageContaining("(role_name)=(Role Name) is not present in table \"roles\"");
```